server/views/pages/overview/partials/workAndInterestsTab/_inductionLongQuestionSet.njk

```diff
@@ -145,11 +147,15 @@ questions are different.
             Specific job role of interest
           </dt>
           <dd class="govuk-summary-list__value">
-            <ul class="govuk-list">
-              {% for specificJobRole in workAndInterests.data.workInterests.longQuestionSetAnswers.specificJobRoles %}
-                <li>{{ specificJobRole }}</li>
-              {% endfor %}
-            </ul>
+            {% if hasAtLeastOneSpecificJobRole %}
```

Comment:
I'm not sure I need this outer if check actually .... I think the array will always have at least 1 element in it, and the nunjucks ternary `{{ job.specificJobRole or 'None' }}` will address it ... 🤔 

Just going to make that change locally and see what happens ....

Reply:
Yup, didn't need it 😁 👍 
Just pushing a commit that removes this outer if statement and simplifies the view object slightly 👍